src/main/common/components/loading-indicator.component.scss

```diff
@@ -0,0 +1,62 @@
+:host {
+  display: block;
+  max-height: 70vh;
+  overflow: auto;
+}
+
+:host-context(.vcd-ext-no-scrolling) {
+  overflow: hidden;
+}
+
+.backdrop {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  position: relative;
+
+  &.backdrop-sm {
+    min-height: .75rem;
+    min-width: .75rem;
+  }
+
+  &.backdrop-md {
+    min-height: 1.5rem;
+    min-width: 1.5rem;
+  }
+
+  &.backdrop-lg {
+    min-height: 3rem;
+    min-width: 3rem;
+  }
+}
+
+:host-context(.vcd-ext-no-scrolling) .backdrop::after {
+  background-color: rgba(#FFF, 0.9);
```

Comment:
They should have. But, the only way to reuse an exising color var is to import "./utils/dependencies.clarity" which i do not want to do now - it will re-import `normalize` and `bootstrap` styles and we will have it twice. I discussed this issue with Clarity few months ago, they are aware, but said it won't be fixed for v1.  In VCD we have custom `dependencies.clarity` which excludes `normalize` and `bootstrap` styles.